content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md

```diff
@@ -44,7 +44,7 @@ As mentioned above, with the promotion of Volume Snapshot to beta, the feature i
 
 In order to use the Kubernetes Volume Snapshot feature, you must ensure the following components have been deployed on your Kubernetes cluster:
 
-- [Kubernetes Volume Snapshot CRDs](https://github.com/kubernetes-csi/external-snapshotter/tree/master/config/crd)
+- [Kubernetes Volume Snapshot CRDs](https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd)
```

Comment:
Another option: link to the tree (by commit ID) at the time of the article. It's OK for blog articles to be a snapshot in time.

Reply:
Great idea @sftim.